release-schema.json

```
@@ -104,6 +103,67 @@
         }
       },
       "minProperties": 1
+    },
+    "Prizes": {
+      "title": "Prizes",
+      "description": "Information about the prizes awarded in the design contest.",
+      "type": "object",
+      "properties": {
+        "description": {
+          "title": "Description",
+          "description": "The description of the prizes.",
+          "type": [
+            "string",
+            "null"
+          ],
+          "minLength": 1
+        },
+        "details": {
+          "title": "Details",
+          "description": "The details of the prizes awarded in the design contest.",
+          "type": "array",
+          "items": {
+            "$ref": "#/definitions/PrizeDetail"
+          },
+          "uniqueItems": true,
+          "minItems": 1
+        }
+      },
+      "minProperties": 1
+    },
+    "PrizeDetail": {
+      "title": "Prize detail",
+      "description": "Details of a prize awarded in the design contest.",
+      "type": "object",
+      "properties": {
+        "id": {
+          "title": "Identifier",
+          "description": "An identifier for the prize.",
```

Per https://ocds-standard-development-handbook.readthedocs.io/en/latest/meta/schema_style_guide.html#identifiers

```suggestion
          "description": "A locally unique identifier for this prize. Used to track changes to this prize and to [merge](https://standard.open-contracting.org/latest/en/schema/merging/#merging) multiple releases to create a record.",
```